content/docs/ops/runbook/rotating-secrets.md

```diff
@@ -0,0 +1,190 @@
+cloud.gov operators in case they are working on a particular piece of software
+that will requires a secrets rotation.
+
+## Rotate Access
+
+It's important that you rotate access to the platform first. This can be
+categorized with three different types of secrets.
+
+1. Remove access to Ops UAA
+1. [Rotate `cg-common` passphrases](#rotate-secrets-passphrases)
+1. [Rotate CF deployment users](#rotate-cf-deployment-users)
+1. [Rotate all IAM Users]({{< relref "docs/ops/runbook/rotating-iam-users.md" >}})
+1. [Rotating Bosh]({{< relref "docs/ops/runbook/rotating-bosh.md" >}})
+1. [Rotating Concourse]({{< relref "docs/ops/runbook/rotating-concourse.md" >}})
+1. [Rotating Cloud Foundry / Diego]({{< relref "docs/ops/runbook/rotating-cloudfoundry-diego.md" >}})
+
+## Rotate CF deployment users
+
+Some of our Concourse pipelines deploy to Cloud Foundry using cloud.gov service
+account deployers. In order to rotate these, you need to target the correct
+org and space in order to create the cloud.gov service deployer account
+instance.
+
+### Auditing all CF deployment users
+
+Use the `get-all-cf-deployers.sh` file in the
+[`cg-scripts`](https://github.com/18F/cg-scripts) repository to audit all of the
+CF users deploying to all environments using Concourse.
+
+The script outputs the login commands for all matching environments for both the
+deployer account and for operators using `--sso`. It's useful to run this
+command redirect the output to a file for the next steps.
+
+When rotating these accounts, you need ensure you don't delete the previous
+deployer service instance _until after_ the credentials have been successfully
+used in a deployment in order to minimize downtime.
+
+### Steps for rotating cloud.gov Service Account deployers
+
+1. Target the appropriate org and space in the current environment.
+1. Check the services, if one exists for this deployer renamed it with a `_prev`
+   suffix for later deletion.
+   - `cf rename-service ${name_of_deployer} ${name_of_deployer}_prev`
+1. Create a new service instance.
+  - `cf create-service cloud-gov-service-account space-deployer "${name_of_deployer}"`
+1. Read the service instance details for the deployer.
+  - `cf service "${name_of_deployer}"`
+1. Open the Fugacious link and save the credentials locally in the `credentials.yml`
+  - `cf service ${name_of_deployer} | grep Dashboard | awk '{ print $2 }'`
+1. If the deployer user is used for more than a `cf push`, give it the proper
+  `cloud_controller.admin` scope manually using `uaac`. Otherwise it will fail
+  in the pipeline job in Concourse. Make sure you authenticate with the correct
+  UAA server when escalating the scopes of the deployer user.
+     - `uaac member add cloud_controller.admin ${guid_of_deployer}`
+1. Refly the pipeline with updated credentials and start a new job.
+1. Delete the `${name_of_deployer}_prev` service account deployer.
+   - `cf delete-service ${name_of_deployer}_prev -f`
+
+## Rotate secrets passphrases
```

Comment:
IMO this *has* to include adding passphrases back to keys in `concourse-credentials`, otherwise we have too much random cleanup to do afterwards.